src/App.tsx

```diff
@@ -1,9 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+export type Movie = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+};
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState<string>('');
+
+  const filterMovies = (list: Movie[]) => {
+    let tmp = [...list];
```

Comment:
[checklist](https://github.com/mate-academy/react_movies-list-filter/blob/master/checklist.md): you don't need to create copy because filter is non-mutating method so it will return altered copy of moviesArray